server/bin/null-basket-server.js

```diff
@@ -95,12 +95,25 @@ function initApp() {
     var params = req.body;
     var email = params.email;
     var user = userData[email];
+    // Basket accepts either an explicit language choice,
+    // or an "accept_lang" preference string from which it
+    // will choose the best language.,
+    var lang = params.lang;
+    if (! lang) {
+      lang = params.accept_lang;
+      if (lang) {
+        lang = lang.split(/[\s\-;,]/)[0];
```

Comment:
So, 'en-US,en;q=0.8' will produce 'en'. Did you mean to split 'en-US'? Shouldn't we just send the first choice locale, and let basket decide if it cares about the country part?